Cargo.toml

```diff
@@ -12,8 +12,8 @@ hyper = "0.10"
 hyper-native-tls = "0.2.1"
 lazy_static = "0.2.0"
 log = "0.3.6"
-r2d2 = "0.7.1"
-r2d2-diesel = "0.16"
+r2d2 = "0.8.2"
+r2d2-diesel = "1.0.0"
```

Comment:
Reading the changelog you linked (super helpful), looks like you may want to double check this: https://github.com/diesel-rs/diesel/blob/master/CHANGELOG.md#added-4